charts/openzaaktypebeheer/values.yaml

```diff
@@ -0,0 +1,199 @@
+  repository: maykinmedia/open-zaaktypebeheer
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  create: true
+  annotations: {}
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext:
+   fsGroup: 1000
+
+securityContext:
+   capabilities:
+     drop:
+     - ALL
+   readOnlyRootFilesystem: false
+   runAsNonRoot: true
+   runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts: []
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 100
+  targetCPUUtilizationPercentage: 80
+  targetMemoryUtilizationPercentage: 80
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+persistence:
+  enabled: true
+  size: 1Gi
+  storageClassName: ""
+  existingClaim: null
+  mediaMountSubpath: openzaaktypebeheer/media
+
+# Existing Secret must be defined for AzureVaultSecret to work
+existingSecret: null
+
+# Django settings
+settings:
+  # -- Generate secret key at https://djecrety.ir/
+  secretKey: ""
+  allowedHosts: ""
+  logStdout: false
+  environment: ""
+  enableAdminNavSidebar: false
+  isHttps: true
+  isDebug: false
+  subpath: null
+
+  cache:
+    default: ""
+    axes: ""
+
+  database:
+    host: ""
+    port: 5432
+    name: ""
+    user: ""
+    password: ""
+
+  email:
+    host: localhost
+    port: 25
+    hostUser: ""
+    hostPassword: ""
+    useTLS: false
+    defaultFrom: ""
+
+  cookie:
+    sessionCookieSamesite: 'Lax'
+    sessionCookieSecure: true
+    csrfCookieSamesite: 'Lax'
+    csrfCookieSecure: true
+
+  oidc:
+    redirectAllowedHosts: ""
+
+  cors:
+    allowedOrigins: ""
+    allowedOriginsRegexes: ""
+    allowAllOrigins: false
+    extraAllowHeaders: ""
+
+  csrf:
+    trustedOrigins: ""
+
+  throttling:
+    enable: true
+    rateAnonymous: "2500/hour"
+    rateUser: "15000/hour"
+
+  twoFactorAuthentication:
+    forceOtpAdmin: true
+    patchAdmin: true
+
+  ui:
+    # The URL where the backend will be running. Needed by the frontend
+    viteBaseApiUrl: ""
+
+nginx:
+  image:
+    repository: nginxinc/nginx-unprivileged
+    pullPolicy: IfNotPresent
+    tag: stable
+  existingConfigmap: null
+  config:
+    clientMaxBodySize: 10M
```

Comment:
It is not being used in the `configmap-nginx`, add it or lose it here. 

If not set you will get the nginx default `1MiB` (uploading files)